### PR TITLE
Dropped Swift 4.2 Support, updated deployment targets

### PR DESCRIPTION
**Pull request by dfuerle**
- Dropped Swift 4.2 Support
- ios depoyment target updated to 10.0
- macOS deployment target updated to 10.12
- tvOS deployment target updated to 10.0
- watchOS deployment target update to 3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +25,21 @@ All notable changes to this project will be documented in this file.
...
+
+#### Updated
+
+- Dropped Swift 4.2 Support
+- ios depoyment target updated to 10.0
+- macOS deployment target updated to 10.12
+- tvOS deployment target updated to 10.0
+- watchOS deployment target update to 3.0
+- Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request
+[#38](https://github.com/Nike-Inc/Elevate/pull/38).
```
Comment (cnoon) [inline CHANGELOG.md]: How about this?

```
#### Updated

- The project to support Swift 5 only and no longer support Swift 4.2.
  - Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
- The deployment targerts to iOS 10.0, macOS 10.12, tvOS 10.0, and watchOS 3.0.
  - Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +25,21 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [4.0.0](https://github.com/Nike-Inc/Elevate/releases/tag/4.0.0)
+
+Released on 2019-04-30. All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/Elevate/milestone/8?closed=1).
```
Comment (cnoon) [inline CHANGELOG.md]: Is the milestone correct here?

**Review by cnoon — CHANGES_REQUESTED** [review]
A couple small changes back to you.

File: CHANGELOG.md (hunk 1 above)
Comment (dfuerle) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (dfuerle) [reply CHANGELOG.md]: it was 10 actually

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +25,18 @@ All notable changes to this project will be documented in this file.
...
+
+Released on 2019-04-30. All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/Elevate/milestone/10?closed=1).
+
+#### Updated
+
+- The project to support Swift 5 only and no longer support Swift 4.2.
+- Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
+- The deployment targerts to iOS 10.0, macOS 10.12, tvOS 10.0, and watchOS 3.0.
+- Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
```
Comment (cnoon) [inline CHANGELOG.md]: Could you add a couple spaces before both of the `- Updated by` lines so they get nested?

**Review by cnoon — CHANGES_REQUESTED** [review]
One last change.

File: .travis.yml [hunk 4]
```diff
@@ -17,11 +17,9 @@ env:
...
     - DESTINATION="OS=11.4,name=iPhone X"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     RUN_TESTS="YES" POD_LINT="NO"
     - DESTINATION="OS=10.3.1,name=iPhone 7"                 SCHEME="$IOS_FRAMEWORK_SCHEME"     RUN_TESTS="YES" POD_LINT="NO"
-    - DESTINATION="OS=9.3,name=iPhone 5S"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     RUN_TESTS="YES" POD_LINT="NO"
-    - DESTINATION="OS=8.4,name=iPhone 5"                    SCHEME="$IOS_FRAMEWORK_SCHEME"     RUN_TESTS="NO"  POD_LINT="NO"
     - DESTINATION="arch=x86_64"                             SCHEME="$MACOS_FRAMEWORK_SCHEME"   RUN_TESTS="YES" POD_LINT="NO"
-    - DESTINATION="OS=12.0,name=Apple TV 4K"                SCHEME="$TVOS_FRAMEWORK_SCHEME"    RUN_TESTS="YES" POD_LINT="NO"
-    - DESTINATION="OS=5.0,name=Apple Watch Series 4 - 44mm" SCHEME="$WATCHOS_FRAMEWORK_SCHEME" RUN_TESTS="NO"  POD_LINT="NO"
+    - DESTINATION="OS=12.2,name=Apple TV 4K"                SCHEME="$TVOS_FRAMEWORK_SCHEME"    RUN_TESTS="YES" POD_LINT="NO"
+    - DESTINATION="OS=5.2,name=Apple Watch Series 4 - 44mm" SCHEME="$WATCHOS_FRAMEWORK_SCHEME" RUN_TESTS="NO"  POD_LINT="NO"
```
Comment (gtrop1) [inline .travis.yml]: Should this be modified to 5.2?

File: CHANGELOG.md [hunk 5]
```diff
@@ -20,6 +25,18 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [4.0.0](https://github.com/Nike-Inc/Elevate/releases/tag/4.0.0)
+
+Released on 2019-04-30. All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/Elevate/milestone/10?closed=1).
```
Comment (eSpecialized) [inline CHANGELOG.md]: move up one line.

File: .travis.yml (hunk 4 above)
Comment (dfuerle) [reply .travis.yml]: yes, so we can test with the latest

**Review by eSpecialized — APPROVED** [review]

**Review by gtrop1 — APPROVED** [review]

**Pull request merged**